src/playback/qml/MuseScore/Playback/internal/MixerTitleSection.qml

```diff
@@ -82,10 +82,12 @@ MixerPanelSection {
             hoverEnabled: true
 
             onContainsMouseChanged: {
-                if (mouseArea.containsMouse) {
-                    ui.tooltip.show(root, channelItem.title)
+                var instrumentName = channelItem.title
+                if (mouseArea.containsMouse && instrumentName.length > 14) {
+                    instrumentName = channelItem.title
+                    ui.tooltip.show(mouseArea, instrumentName)
```

Comment:
Let's eliminate the `instrumentName` variable, since it's written and read only once:
```suggestion
                if (mouseArea.containsMouse && instrumentName.length > 14) {
                    ui.tooltip.show(mouseArea, channelItem.title)
```